packages/grid/_modules_/grid/hooks/features/export/useGridCsvExport.tsx

```diff
@@ -22,13 +22,19 @@ export const useGridCsvExport = (apiRef: GridApiRef): void => {
     return buildCSV(visibleColumns, visibleSortedRows, selection, apiRef.current.getCellValue);
   }, [logger, visibleColumns, visibleSortedRows, selection, apiRef]);
 
-  const exportDataAsCsv = React.useCallback((): void => {
-    logger.debug(`Export data as CSV`);
-    const csv = getDataAsCsv();
-    const blob = new Blob([csv], { type: 'text/csv' });
+  const exportDataAsCsv = React.useCallback(
+    (options?: GridExportCsvOptions, fileName?: string): void => {
+      logger.debug(`Export data as CSV`);
+      const csv = getDataAsCsv();
 
-    exportAs(blob, 'csv', 'data');
-  }, [logger, getDataAsCsv]);
+      const blob = new Blob([options?.utf8WithBom ? new Uint8Array([0xef, 0xbb, 0xbf]) : '', csv], {
```

Comment:
can we abstract that? It doesn't need to be in the hook

Reply:
Do you mean to make this line as external function?